extract/raw_array_cohort_extract.wdl

```diff
@@ -0,0 +1,229 @@
+version 1.0
+
+workflow RawArrayCohortExtract {
+   input {
+        Int number_of_partitions = 2
+        Int probes_per_partition = 1000000
+        
+        File reference
+        File reference_index
+        File reference_dict
+    
+        String? fq_probe_info_table
+        File? probe_info_file
+        
+        String fq_dataset
+        Int max_tables
+        String fq_destination_dataset
+        String query_project
+        String fq_cohort_mapping_table
+        Int ttl = 24
+        
+        String output_file_base_name
+        
+        String full_sample_info_table
+    }
+    
+    call CreateExtractTable {
+        input:
+            fq_dataset                = fq_dataset,
+            max_tables                = max_tables,
+            fq_destination_dataset    = fq_destination_dataset,
+            query_project             = query_project,
+            fq_cohort_mapping_table   = fq_cohort_mapping_table,
+            ttl                       = ttl,
+            number_of_partitions      = number_of_partitions,
+            probes_per_partition      = probes_per_partition
+    }
+  
+    
+    scatter(i in range(number_of_partitions)) {
+        call ExtractTask {
+            input:
+                reference             = reference,
+                reference_index       = reference_index,
+                reference_dict        = reference_dict,
+                fq_probe_info_table   = fq_probe_info_table,
+                probe_info_file       = probe_info_file,
+                min_probe_id          = 1 + i * probes_per_partition,
+                max_probe_id          = (i+1) * probes_per_partition,
+                sample_info_table     = full_sample_info_table,
+                use_compressed_data   = "false",
+                cohort_extract_table  = CreateExtractTable.cohort_extract_table,
+                project_id            = query_project,
+                output_file           = "${output_file_base_name}_${i}.vcf.gz"
+        }
+    }
+
+    call MergeVCFs { 
+       input:
+           input_vcfs = ExtractTask.output_vcf,
+           input_vcfs_indexes = ExtractTask.output_vcf_index,
+           output_vcf_name = "${output_file_base_name}.vcf.gz",
+           preemptible_tries = 3
+    }
```

Comment:
Can you add an output block in case we ever run this in Terra?

```suggestion
    }
    output {
        File output_vcf = MergeVCFs.output_vcf
        File output_vcf_idx = MergeVCFs.output_vcf_index
    }
```

Reply:
👍